baseband/mark5b/base.py

```diff
@@ -158,29 +159,34 @@ class Mark5BStreamReader(VLBIStreamReaderBase, Mark5BFileReader):
         Reference MJD (rounded to thousands), to remove ambiguities in the
         time stamps.  By default, will be inferred from the file creation date.
     thread_ids: list of int, optional
-        Specific threads to read.  By default, all threads are read.
+        Specific channels to read.  By default, all channels are read.
     frames_per_second : int, optional
         Needed to calculate timestamps. If not given, will be inferred from
         ``sample_rate``, or by scanning the file.
     sample_rate : `~astropy.units.Quantity`, optional
         Rate at which each thread is sampled (bandwidth * 2; frequency units).
+    squeeze : bool
+        If `True` (default), remove channel and thread dimensions if unity.
     """
 
     _frame_class = Mark5BFrame
```

Comment:
Maybe add `_payload_class` or use `self._frame_class._payload_class._sample_shape_cls(...)`

Is `_frame_class` even used? If so, shouldn't it be in the binary reader?

Reply:
See separate issue #87